packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -769,7 +770,10 @@ export class CdkToolkit {
       defaultBehavior: DefaultSelection.OnlySingle,
     });
 
-    // No validation
+    const notExistPatterns = selector.patterns.filter(p => !stacks.stackArtifacts.find(s => minimatch(s.hierarchicalId, p)));
```

Comment:
Could you also add code such that we error out before it we even try deleting any stack, if at least one of the stacks provided does not exist?

Reply:
@SankyRed 

In the current code, if any of the stacks are not existing, none of the stacks are deleted. And the cdk occurs an error.

Is this not what you are assuming?